files/system/usr/share/ublue-os/just/60-custom.just

```diff
@@ -559,3 +562,27 @@ audit-secureblue:
         done
     fi
 
+    GHNS_TEST_STRING="Ensuring KDE GHNS is disabled"
+    KDE_GLOBALS_FILE="/etc/xdg/kdeglobals"
+    if test -e $KDE_GLOBALS_FILE; then
+        if grep -q "ghns=false" "$KDE_GLOBALS_FILE"; then
+            print_status "$GHNS_TEST_STRING" "$STATUS_SUCCESS"
+        else
+            print_status "$GHNS_TEST_STRING" "$STATUS_FAILURE"
+        fi
+    fi
+
+    HARDENED_MALLOC_TEST_STRING="Ensuring hardened_malloc is set in ld.so.preload"
+    if grep -qw "libhardened_malloc.so" "/etc/ld.so.preload"; then
```

Comment:
this will succeed even if someone comments out libhardened_malloc.so like:

```
# libhardened_malloc.so
```

instead, you can simply diff `/usr/etc/ld.so.preload` and `/etc/ld.so.preload` and fail if there's a difference. check the modprobe blacklist test for an example